__tests__/pages/signup.js

```diff
@@ -1,29 +1,198 @@
 import React from 'react'
 import { render, fireEvent, wait } from '@testing-library/react'
 import Signup from '../../pages/signup'
+import * as signupHelper from '../../helpers/signupUser'
+import Router from 'next/router'
 
 describe('Signup Page', () => {
-  const submitSignup = jest.fn()
-  const props = { submitSignup }
-
   test('Should render without crashing', () => {
-    const { getByTestId } = render(<Signup {...props} />)
+    const { getByTestId } = render(<Signup />)
     getByTestId('email')
     getByTestId('username')
     getByTestId('password')
     getByTestId('firstName')
     getByTestId('lastName')
   })
 
-  test('Should not submit values', async () => {
-    const { getByTestId } = render(<Signup {...props} />)
+  //to be readded once validation of form takes place in real time
```

Comment:
Please update description so nobody else gets confused like you did.
`Should not submit values if nothing was typed in`